src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -978,18 +992,37 @@ public function get_source_info() {
 	 * Fetches the status message for the last import attempt on (scheduled) records
 	 *
 	 * @param string $type Type of message to fetch
+	 * @param bool   $lookup_children Whether the function should try to read the last children post status to return a coherent
+	 *                                last import status or not, default `false`.
 	 *
-	 * @return string
+	 * @return bool|string Either the message corresponding to the last import status or `false` if the last import status
+	 *                     is empty or not the one required.
 	 */
-	public function get_last_import_status( $type = 'error' ) {
+	public function get_last_import_status( $type = 'error', $lookup_children = false ) {
 		$status = empty( $this->meta['last_import_status'] ) ? null : $this->meta['last_import_status'];
 
+		if ( empty( $status ) && $lookup_children ) {
+			$last_children_query = $this->query_child_records( array( 'posts_per_page' => 1, 'order' => 'DESC', 'order_by' => 'modified' ) );
+			if ( $last_children_query->have_posts() ) {
+				$last_children        = reset( $last_children_query->posts );
+				$last_children_status = str_replace( 'tribe-ea-', '', $last_children->post_status );
```

Comment:
There is no need to do this if we pass `tribe-ea-failed` and `tribe-ea-success` as the mapping strings.